scenarios7/14_Arctic_Wastelands.cfg

```diff
@@ -962,47 +962,7 @@ You can build a keep anywhere for 50 gold, or a village for 10 gold. (see option
             experience=no
         [/kill]
     [/event]
-    [event]
-        name=attacker_misses
-        first_time_only=no
-        [filter_second]
-            ability=lethalia_evasion
-        [/filter_second]
-        [store_locations]
-            [filter_adjacent_location]
-                x,y=$x2,$y2
-            [/filter_adjacent_location]
-            [not]
-                terrain=Xuc,Xu,Xol,Xos,Zic,Ms^Xm
-            [/not]
-            variable=dodge_locs
-        [/store_locations]
-        [foreach]
-            array=dodge_locs
-            [do]
-                [if]
-                    [have_unit]
-                        x,y=$this_item.x,$this_item.y
-                    [/have_unit]
-                    [else]
-                        {MOVE_UNIT id=$second_unit.id $this_item.x $this_item.y}
-                        [break][/break]
-                    [/else]
-                [/if]
-            [/do]
-        [/foreach]
-        {CLEAR_VARIABLE dodge_locs}
-    [/event]
-    [event]
-        name=attacker_misses
-        [filter_second]
-            ability=lethalia_evasion
-        [/filter_second]
-        [message]
-            speaker=second_unit
-            message= _ "Hahaha, you still think you can match my dexterity?"
-        [/message]
-    [/event]
+    {EVASION lethalia_evasion "Xuc,Xu,Xol,Xos,Zic,Ms^Xm"}
```

Comment:
Can you just put all these hex types into the event and don't pass it as an argument? I think the extra terrains that would match the filter would not change anything.

Reply:
Yes, but Uria's look pretty different from the rest.  Would "Ms^Xm,Q*,X*,Zic" work for everything?

Reply:
Yes, I am quite sure these terrains don't appear around the evil Lethalia.